rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md

```diff
@@ -3,7 +3,7 @@ type: rule
 archivedreason: 
 title: Do you have flexibility with the order you do the work (i.e. cancelling a Sprint)?
 guid: d98a0462-7682-4532-b440-5376d2d84a44
-uri: are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint
+uri: unexpected-requests
```

Comment:
Need to change the file name as well

Reply:
I assume you mean the folder? I've done that - give it a check and resolve if you're happy.

Reply:
yep folder!